src/js/select.js

```diff
@@ -286,10 +335,14 @@ Menu.prototype.clickHandler = function(ev) {
 Menu.prototype.increment = function() {
   if (this.currentIndex === this.menuEl.children.length - 1) return;
 
+  // These can be both options and optgroups.
   var optionEls = this.menuEl.children;
   
   jqLite.removeClass(optionEls[this.currentIndex], selectedClass);
```

Comment:
You can save multiple hash table lookups by caching the result of `optionEls[this.currentIndex]`